src/stactools/sentinel1/rtc_metadata.py

```diff
@@ -147,11 +149,14 @@ def epsg(self) -> Optional[str]:
     def metadata_dict(self):
         ''' match s2 l2a cogs from https://earth-search.aws.element84.com/v0 '''
         sentinel_metadata = {
+            # NOTE: once PySTAC adds MGRS extension can move this to stac.py
+            'mgrs:utm_zone': self.metadata['TILE_ID'][:2],
+            'mgrs:latitude_band': self.metadata['TILE_ID'][2],
+            'mgrs:grid_square': self.metadata['TILE_ID'][3:],
             'sentinel:mgrs': self.metadata['TILE_ID'],
```

Comment:
keeping this as a property b/c was unsure how to merge multiple `mgrs:properties` when creating a test catalog https://github.com/stac-utils/pystac/issues/480